deploy/helm/sumologic/values.yaml

```diff
@@ -248,6 +248,8 @@ fluentd:
   securityContext:
     ## The group ID of all processes in the statefulset containers. By default this needs to be fluent(999).
     fsGroup: 999
+    runAsUser: 0
+    runAsGroup: 0
```

Comment:
Why are we making this change?

Reply:
This is a test 1,2,3.
Thanks for staying vigilant! 🤦